src/Microsoft.Framework.Logging.Abstractions/LoggerExtensions.cs

```diff
@@ -581,17 +575,17 @@ public static void LogCritical([NotNull] this ILogger logger, int eventId, strin
             ILogValues state,
             Exception exception = null)
         {
-            logger.Log(logLevel, eventId, state, exception, _logValuesFormatter);
+            logger.Log(logLevel, eventId, state, exception, MessageFormatter);
         }
 
-        private static string LogValuesFormatter(ILogValues state, Exception exception)
+        private static string MessageFormatter(object state, Exception exception)
         {
             if (exception == null)
             {
                 return state?.ToString();
             }
 
-            return state?.ToString() + Environment.NewLine + exception;
+            return string.Format(CultureInfo.CurrentCulture, "{0}{1}{2}", state, Environment.NewLine, exception);
```

Comment:
> I bet the typical quick & dirty logging pattern is to just have a log level and a message.

Here `state` represents the message actually...so did you mean to say 'is to just have a log level and an exception'?